script/Ciel/Day 3/03_01_CIEL03_3_2B.txt

```diff
@@ -27,9 +27,8 @@ A low droning hiss echoes in my ears.
 [sha:94823b2e96f953fa9db962b1239083e01152450d]{
 -- Page 129, Offset 5169.
 -- ざあ。ざあ。ざあ。
-// Pitter patter feels too discrete, but there's no great en equivalent for the
-// ザーーー of steady curtains of rain
-%{i}Fshhhhhhhh%{/i}.
+// comes back in 5173
+%{i}Pitter, patter.%{/i}.
```

Comment:
Probably.

Reply:
Pitter patter is kinda the wrong onomatopoeia here imo. It's discrete and light. I would probably change to just 
> %{i}Hisssss%{/i}

to echo prev if we want to keep ono, but given the relative prevalence in each language I think it may be more stylish to replace it like we did later.
> Steady curtains of rain batter the ground.